src/Microsoft.ML.Core/Utilities/DoubleParser.cs

```diff
@@ -12,26 +12,26 @@ namespace Microsoft.ML.Internal.Utilities
     [BestFriend]
     internal static class DoubleParser
     {
+        [BestFriend]
+        [Flags]
+        internal enum OptionFlags : uint
+        {
+            Default = 0x00,
+
```

Comment:
suggestion: OptionFlags.Default does not carry much meaning. Please consider an explicit UsePeriodAsDecimalMarker

Reply:
It's easier to have this option, so that I can easily check if we're going to have default OptionFlags on the TextLoader.Parser, and then branch accordingly to create a custom instance or not. I also decided to have it, since the default value for the overloads for TryParse(), Parse(), Conversion() and ValueCreatorCache(), all need to have a default flag to work correctly, and I found it more legible to use OptionFlags.Default.
Also, as new options are added, then Default would have more meaning.

On the other hand, adding a UsePeriodAsDecimalMarker won't make much sense, since it will never be the case that we're using both , and . as decimal markers. So it would be redundant to have this flag since we already have the UseCommaAsDecimalMarker flag.

---
In reply to: [429586665](https://github.com/dotnet/machinelearning/pull/5154#discussion_r429586665) [](ancestors = 429586665)